### PR TITLE
Add Kmail as sieve client

### PR DESCRIPTION
**Pull request by pbek**
# Description

The sieve client of [Kmail](https://kontact.kde.org/components/kmail.html) works great with docker-mailserver, I propose to add it as sieve client to the documentation.

## Type of change

- [x] Improvement (non-breaking change that does improve existing functionality)
- [x] This change requires a documentation update

## Checklist:

No code, just a documentation update.

### CODE REVIEW

File: docs/content/config/advanced/mail-sieve.md [hunk 1]
```diff
@@ -83,3 +83,4 @@ All user defined sieve scripts that are managed by ManageSieve are stored in the
 The extension is known to work with the following ManageSieve clients:
 
 - **Sieve Editor**  a portable standalone application based on the former Thunderbird plugin (https://github.com/thsmi/sieve).
```
Comment (wernerfred) [inline docs/content/config/advanced/mail-sieve.md:85]:
```suggestion
- **[Sieve Editor](https://github.com/thsmi/sieve)**  a portable standalone application based on the former Thunderbird plugin.
```

File: docs/content/config/advanced/mail-sieve.md [hunk 2]
```diff
@@ -83,3 +83,4 @@ All user defined sieve scripts that are managed by ManageSieve are stored in the
...
+- **Kmail**  the mail client of [KDE](https://kde.org/)'s Kontact Suit (https://kontact.kde.org/components/kmail.html).
```
Comment (wernerfred) [inline docs/content/config/advanced/mail-sieve.md:86]:
```suggestion
- **[Kmail](https://kontact.kde.org/components/kmail.html)**  the mail client of [KDE](https://kde.org/)'s Kontact Suite.
```

**Review by wernerfred — DISMISSED** [review]
Only stylistic request: Can you please change the linking to the one i suggested?
Otherwise LGTM

**Comment by pbek** [comment]
Sure, thank you. I just tried to make it look like the other link. 😉  (and thank you for the typo fix)

**Review by polarathene — APPROVED** [review]
Thanks for the contribution @pbek :smiley: 

Big fan of QOwnNotes btw!

**Comment by pbek** [comment]
Haha, thank you very much, @polarathene. 😁

**Pull request merged**